### PR TITLE
Script for migrating Ratings table.

### PR DESCRIPTION
**Pull request by jakebarnett**

### CODE REVIEW

File: src/main/scala/com/apptentive/ratings/MigrateRatings.scala [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    for (app <-listOfAppIds) {
+      val batch = new BatchStatement
+
+      val store = if (app.split(',')(0) == "iOS") "itunes" else "android"
+      val appId = app.split(',')(1)
+
+      val sourceTable = args(1)
+      val v1AppRatings: ResultSet = session.execute(s"select * from $sourceTable where store_app_id = '$appId' and store ='$store'")
+
+      v1AppRatings.all.foreach(row => {
```
Comment (KonstantinTsykulenko) [inline src/main/scala/com/apptentive/ratings/MigrateRatings.scala]: Won't "all" fetch all results at once? I think we would want to use a cursor,

Comment (KonstantinTsykulenko) [inline src/main/scala/com/apptentive/ratings/MigrateRatings.scala]: Are we loading the whole table in memory here? I don't see any limits in the select statement before, and all() should load everything into memory

Comment (KonstantinTsykulenko) [inline src/main/scala/com/apptentive/ratings/MigrateRatings.scala]: Have you checked how many ratings we can potentially have per app?

Comment (jakebarnett) [reply src/main/scala/com/apptentive/ratings/MigrateRatings.scala]: We get 1 rating per day back to 2012-01-01, so ~2,273 max.  I wouldn't imagine that would be too much to pull into memory.

**Pull request merged**